Magitek/Logic/Summoner/Aoe.cs

```diff
@@ -203,15 +204,15 @@ public static async Task<bool> Outburst()
             {
                 target = Combat.SmartAoeTarget(Spells.BrandofPurgatory, SummonerSettings.Instance.SmartAoe);
 
-                if (target == null)
+                if (target == null || Core.Me.CurrentTarget == null)
                     return false;
                 
                 return await Spells.BrandofPurgatory.Cast(target);
             }
 
             target = Combat.SmartAoeTarget(Spells.PreciousBrilliance, SummonerSettings.Instance.SmartAoe);
             
-            if (target == null)
+            if (target == null || Core.Me.CurrentTarget == null)
```

Comment:
This seems weird to me, since the code is explicitly trying to find an AOE target, I don't know if we should return false if our current target is null? I guess it is weird to AOE when we don't have anything targeted. 

If you think this is right I'm good with it, but it seems odd to me in any case.